File: tts/tt02/cometa/Cometa.java
import java.util.*;

public class Cometa
{
    static Scanner sc = new Scanner(System.in);

    public static int getYear(int year)
    {
        int rem = (year - 1986) % 76;

        if (rem == 0)
        {
            return (year + 76);
        }
        else
        {
            return (year + 76 - rem);
        }
    }

    public static void main(String[] args)
    {
        int year = sc.nextInt();
        int next;

        while (year != 0)
        {
            next = getYear(year);
            System.out.println(next);
            year = sc.nextInt();
        }
    }
}

File: tts/tt02/cometa/cometa.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int getYear(int year)
{
    int rem = (year - 1986) % 76;

    if (rem == 0)
    {
        return (year + 76);
    }
    else
    {
        return (year + 76 - rem);
    }
}

int main(void)
{
    int year;
    int next;

    scanf("%d", &year);

    while (year != 0)
    {
        next = getYear(year);
        printf("%d\n", next);
        scanf("%d", &year);
    }
}
